keras/ops/core_test.py

```diff
@@ -529,14 +534,23 @@ def log1pexp_nan(x):
                 z = log1pexp_nan(x)
             dy_dx = tape1.gradient(y, x)
             dz_dx = tape2.gradient(z, x)
+            self.assertEqual(ops.convert_to_numpy(dy_dx), 1.0)
         elif backend.backend() == "jax":
             import jax
 
             dy_dx = jax.grad(log1pexp)(x)
             dz_dx = jax.grad(log1pexp_nan)(x)
+            self.assertEqual(ops.convert_to_numpy(dy_dx), 1.0)
+            self.assertTrue(ops.isnan(dz_dx))
+        elif backend.backend() == "torch":
+            import torch
+
+            x = torch.tensor(100.0, requires_grad = True) # x = ops.convert_to_tensor(100.0) is NOT supported Yet!
```

Comment:
Hello, @james77777778 @fchollet I just found out why PyT unit tests for `@custom_gradient` were failing all along.

The problem is that if one wants to calculate or work with gradients of a variable in torch, then regular definition of that variable by tensor `torch.tensor(100.0)` doesn't works. It needs additional `torch.tensor(100.0, requires_grad = True)` argument to be able to have `.grad` instance variable later on.

Keras' `ops.convert_to_tensor()` too requires this argument to define tensor in PyTorch using `requires_grad` argument to be able to calculate the gradient.

This is why I had to manually use PyT here to define `torch.tensor` rather than to have `ops.convert_to_tensor` to make it work.

Reply:
You can instead do:

```python
x = keras.Variable(100.0).value
```

This will work with all backends.